### DISCUSSION TITLE
Per-Service or Host notes_url_target or action_url_target

### DISCUSSION THREAD

**Discussion by LaFong** [sni/thruk#1367] [Q&A] [answered]
Feature request: Currently, you can only set an overall notes_url_target or action_url_target in cgi.cfg. For some of our notes_url's we'd like "_blank", others should be "main". It would be nice to have per-service and per-host directives to do this.

**Comment by sni** [comment, marked as answer]
Not exactly what you want, but might solve the issue as well:
https://thruk.org/documentation/configuration.html#_link_target
This option sets the url target base on the url itself. Usually the setup looks like for example all doc links should open in a new window while graphing links open in the main frame. And that's way easier to configure based on url pattern.

**Reply by LaFong** [reply]
Wonderful, that works fine, don’t know how I missed it.
